content/blog/hello-world.md

```diff
@@ -1,28 +1,41 @@
 +++
 title = "Hello world"
-date = 2022-01-19T14:28:05+01:00
+date = 2022-04-04T14:28:05+01:00
 description = "Introducing scverse"
 draft = false
 +++
 
-With the increasing reliance of scientific research on infrastructural and analytical software, foundational tools in life sciences have outgrown individual labs and institutes in terms of their scale and impact. That has encouraged us to start something more than a code repository for an exciting new project – we are starting a new organisation to maintain and support foundational tools in the life sciences.
+Due to the increasing reliance of scientific research on complex computational pipelines, foundational software in the life sciences has outgrown individual labs and institutes in terms of its scale and impact. This has motivated us to start something more than a code repository for an exciting new project – a new organisation to maintain and support foundational tools in the life sciences in the Python ecosystem. We are therefore very excited to announce **scverse** – a consortium for joint development of major packages in omics data analysis in life sciences.
```

Comment:
```suggestion
Due to the increasing reliance of scientific research on complex computational pipelines, foundational software in the life sciences has outgrown individual labs and institutes in terms of its scale and impact.
This has motivated us to start something more than a code repository for an exciting new project – a new consortium to organize and support core (mostly) Python tools for single cell life science research.
We are excited to announce **scverse**.
```

Reply:
Our slogan is `Foundational tools for omics data in the life sciences` -> I'd prefer tools over software here? The rest of your changes I agree with

Reply:
Main point is that this:

Due to the increasing reliance of scientific research on complex computational pipelines, **`foundational software in the life sciences`** has outgrown individual labs and institutes in terms of its scale and impact. This has motivated us to start something more than a code repository for an exciting new project – a new organisation to maintain and support **`foundational tools in the life sciences`** in the Python ecosystem. We are therefore very excited to announce **scverse** – a consortium for joint development of **`major packages in omics data analysis in life sciences`**.

is too much

Idk that we need to even say the exact thing once.

Reply:
Ahh, ouch yeah. Too tired to notice. Feel free to fix and commit. You have my sign of approval :)

Reply:
🙏